renderer.js

```diff
@@ -985,14 +1007,72 @@ function playPressed(){
 
         playModalQuitDLCancel = dataObj.reqObj;
         
+        
     });
 
 }
 
-playButtonText.addEventListener("click", function(event){
+async function checkIfCompatible() {
+    try {
+        const data = await si.graphics();
+
+        // Get all of the graphics cards info
+        var i;
+        for (i = 0; i < data.controllers.length; i++){
+            graphicControllers = data.controllers[i];
+            cards = String(graphicControllers.model);
+            console.log(cards);
+        }
+
+        // Is incompatible if the word Intel is found in a substring
+        if(cards.includes("Intel")){
+            showIncompatiblePopup = true;
+        }
+    } catch (e) {
+        console.log(e)
+    }
+}
 
+playButtonText.addEventListener("click", function(event){
     console.log("play clicked");
-    playPressed();
+
+    playModal.show();
+
+    let playBox = document.getElementById("playModalContent");
+    playBox.innerHTML = "<p id='playingInternalP'></p>"
+
+    let status = document.getElementById("playingInternalP");
+
+    if(showIncompatiblePopup){
+        status.textContent = "Detected graphics card(s): " + cards;
+        status.append(document.createElement("br"));
+        status.append(document.createElement("br"));
+        status.append(document.createTextNode("WARNING: Intel Integrated Graphics Card may causes Thrive to crash due to graphics engine issues with Intel: https://github.com/Revolutionary-Games/Thrive/issues/804."))
+        status.append(document.createElement("br"));
+        status.append(document.createTextNode("This is a known problem, any help fixing this would be most welcome!"));
+
+        let closeContainer = document.createElement("div");
```

Comment:
If the detected graphics has "nvidia" or "amd" in it an additional help text telling the user to configure thrive to run with that graphics card, would be helpful.